src/components/Tooltip/index.js

```diff
@@ -1,20 +1,22 @@
 // @flow
 import React from 'react';
-import TooltipHolder from './TooltipHolder';
-import TooltipInfo from './TooltipInfo';
-import Tipneed from './Tipneed';
+import TooltipWrapper from './TooltipWrapper';
 
 type Props = {
-  info: object,
+  spanText: object,
+  placement: string,
+  dataTip: string,
 };
 
-const Tooltip = ({ info }: Props) =>
-  info ? (
-    <TooltipHolder>
-      <Tipneed>
-        <TooltipInfo>{info}</TooltipInfo>
-      </Tipneed>
-    </TooltipHolder>
+const Tooltip = ({ spanText, placement, dataTip }: Props) =>
```

Comment:
Instead of sending a `spanText` prop we can use the `children` prop to make it easier to use, in my opinion:
```suggestion
const Tooltip = ({ placement, dataTip, children }: Props) =>
```

And to use it:
```jsx
<Tooltip dataTip="Some tip" placement="top">
  Hover me
</Tooltip>
```